src/main/java/org/session5/homework/FibonacciSeriesGenerator.java

```diff
@@ -0,0 +1,27 @@
+package org.session5.homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FibonacciSeriesGenerator {
+
+    private FibonacciSeriesGenerator() {
+        throw new IllegalStateException("Fibonacci Series Generator");
```

Comment:
Your private constructor throws an exception with the message "Fibonacci Series Generator". It's a good practice to prevent the instantiation of utility classes, but the message should be more descriptive, like "Utility class should not be instantiated."